### PR TITLE
feat: use Loadable in ChartGrid and LineChart

### PR DESCRIPTION
**Pull request by johnkim-det**
## Description

Changes:
`<ChartGrid>`'s `chartsProps` prop can optionally be a `Loadable` value. 
`<LineChart>`'s `series` prop can optionally be a `Loadable` value.

Displaying appropriate Empty/Loading states for these components, with examples added to UI kit page.

## Test Plan

Verify that UI kit page displays new states in the "Charts" section.

## Checklist

- [ ] Changes have been manually QA'd
- [ ] User-facing API changes need the "User-facing API Change" label.
- [ ] Release notes should be added as a separate file under `docs/release-notes/`.
  See [Release Note](https://github.com/determined-ai/determined/blob/master/docs/release-notes/README.md) for details.
- [ ] Licenses should be included for new code which was copied and/or modified from any external code.

## Ticket

WEB-1003

### CODE REVIEW

File: webui/react/src/components/kit/LineChart.tsx [hunk 1]
```diff
@@ -323,18 +337,34 @@ export const ChartGrid: React.FC<GroupProps> = React.memo(
             <XAxisFilter options={xAxisOptions} value={xAxis} onChange={onXAxisChange} />
```
Comment (mapmeld) [inline webui/react/src/components/kit/LineChart.tsx:337]: The ChartGrid's `ScaleSelect` above should also be hidden when empty or isLoading

**Review by ClaireNeveu — COMMENTED** [review]
For the single charts the background doesn't extend over the empty state.
Screen Shot 2023-03-15 at 17 00 08

**Comment by ClaireNeveu** [comment]
For some reason there's an empty SVG in the "empty" case pushing the text down.

**Comment by mapmeld** [comment]
Thanks for fixing re: previous comment
I think the blank SVG issue on the single chart is from MessageType.Empty - It's used in a few other places.

**Comment by ClaireNeveu** [comment]
Regardless of why it appears, if I delete it from the DOM the component looks good (text is centered).

**Comment by johnkim-det** [comment]
addressing the svg issue by removing `<Message>` from `<UPlotChart>`. also removing the `noDataMessage` prop.

**Review by mapmeld — APPROVED** [review]
✅ 📈  LGTM

**Review by ClaireNeveu — APPROVED** [review]
Design looks good!

**Pull request merged**